frontend/src/employee-frontend/components/messages/MessageEditor.tsx

```diff
@@ -285,8 +306,7 @@ export default React.memo(function MessageEditor({
     [nestedAccounts, availableReceivers, selectedUnit.value]
   )
 
-  const sendEnabled =
-    !sending && draftState === 'clean' && areRequiredFieldsFilled(message)
```

Comment:
Why was the `draftState === 'clean'` predicate removed?

Reply:
I wonder if it makes sense to require the user to wait for clean state before send, since the draft will be deleted anyway.

Reply:
> Or is this required because of the attachments?

Basically yes. Otherwise they may be out of sync.